### PR TITLE
Re-enable swiftformat and swiftlint

### PR DESCRIPTION
**Pull request by ipavlidakis**
### 🎯 Goal

Enable again swift linter and formatter in order to improve consistency in the codebase and correct common stylin issues before a PR.

### ☑️ Contributor Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] This change follows zero ⚠️ policy (required)
- [x] This change should receive manual QA
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (Docusaurus, tutorial, CMS)

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -581,11 +582,11 @@ lane :rubocop do
...
 end
 
-desc 'Run source code linting'
-lane :run_linter do
+desc 'Run source code formatting/linting'
+lane :run_swift_format do |options|
   Dir.chdir('..') do
-    UI.error('SwiftFormat lint was skipped. Check out https://github.com/GetStream/ios-issues-tracking/issues/465 for more info')
-    # sh('mint run swiftformat --lint --config .swiftformat Sources --exclude **/Generated, **/generated, **/protobuf, **/OpenApi, Sources/StreamChatSwiftUI/StreamNuke')
+    action = options[:lintOnly] ? "--lint" : ""
```
Comment (testableapple) [inline fastlane/Fastfile:588]: No camel case in Ruby plz 🙈
```suggestion
    action = options[:lint_only] ? "--lint" : ""
```


Or even just `lint`, the other command will just format without linting, won't it?
```suggestion
    action = options[:lint] ? "--lint" : ""
```

File: fastlane/Fastfile [hunk 2]
```diff
@@ -581,11 +582,11 @@ lane :rubocop do
...
+    sh("mint run swiftformat #{action} --config .swiftformat --exclude #{swiftformat_excluded_paths.join(",")} Sources")
```
Comment (testableapple) [inline fastlane/Fastfile:589]:
```suggestion
    sh("mint run swiftformat #{action} --config .swiftformat --exclude #{swiftformat_excluded_paths.join(',')} Sources")
```

**Review by testableapple — APPROVED** [review]
Awesome work! 🚢

Just a QQ, didn't you update the pre-commit hook?

**Comment by ipavlidakis** [comment]
@testableapple i updated the pre-commit.sh that runs as part of bootstrap. You will need to run bootstrap to get the new pre-commit

**Comment by martinmitrevski** [comment]
If I remember correctly - we disabled it back then because swiftformat modified the code into a non-compilable one. Since the checks now pass, I guess this is no longer an issue.

**Comment by testableapple** [comment]
Yeah, just my GitHub page showed there were no changes in `pre-commit.sh`, but now all good 👍

**Comment by ipavlidakis** [comment]
> If I remember correctly - we disabled it back then because swiftformat modified the code into a non-compilable one. Since the checks now pass, I guess this is no longer an issue.

I actually fixed the issue and the project runs correctly locally. Even on the previous checks it was compiling and running all tests successfully.

**Pull request merged**